### DISCUSSION TITLE
I have student email id that colllege provided me but still my github pack is  not approved

### DISCUSSION THREAD

**Discussion by abhinav10130** [community/community#131958] [GitHub Education]
### Hi everyone,

Hi everyone.  my student email id ends with .ac.in not with .edu. so i am keep getting rejected till this date i rejected 15 times . even i have taken the live photo of my Student ID  , college fee transcript but still its not working. please help me and tell me what to do and how can i contact support .  and while uploading the live photo i am not able to use my back camera of my Device. so caputuring photo with a front camera is difficult . even i have added my college email id thats end with domain .ac.in but still in the rejection mail i am getting a error that "Please use your academic email"

**Comment by krikera** [comment]
You should also add your college email into your own personal github account or the account you want to use.

**Reply by abhinav10130** [reply]
I have already added my college email id on my personal  github account

**Comment by Aditijainnn** [comment]
change username on  profile to real name.
    change location on profile to your college located prefecture instead of your house.
    also Add your school/college email into your own personal github account or the account you want to use.
    after updating profile, sign out from GitHub.
    after for a while, sign in again and resend application

**Reply by abhinav10130** [reply]
I did that but still not working.  even i tried to take a photo of my student email id when i was in college but its not working . i have my real name on my github profile and billings

**Reply by Aditijainnn** [reply]
Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted.
would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

**Comment by samus-aran** [comment]
Hi @abhinav10130 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

As that school prefers an academic email for verification I would recommend you reach out and provide evidence that the student domain is inaccurate or needs updating.
